scripts/protoc-gen-depends

```diff
@@ -81,4 +88,4 @@ def add_depends(fd):
 
 res.file.extend([add_depends(fileDescriptor) for fileDescriptor in req.proto_file])
 
-stdout.write(res.SerializeToString())
```

Comment:
And here probably `decode('utf-8`)`. Have not tried this but may give the same results.